### PR TITLE
added winter members (w/o photos)

### PR DESCRIPTION
**Pull request by AVDestroyer**

### CODE REVIEW

File: data/members.ts [hunk 1]
```diff
@@ -7,419 +7,335 @@ export type PersonInfoProps = {
...
+  },
   {
     name: "Benson Liu",
-    role: "Co-President, Psi Beta Rho Director",
+    role: "Advisor",
     bio: "Hey, I'm Benson! I'm a 3rd year Computer Science major from New Jersey who enjoys hacking & playing CTFs! In my free time, I like to watch anime or baseball (my favorite team is the Yankees).",
     pronouns: "he/him",
-    photo: "bliu.png",
+    photo: "bliu.png"
   },
```
Comment (r2dev2) [inline data/members.ts:30]: Can you move Benson and Benson only down 🤡.

His lengthy bio causes the whole next row of officers to be yeeted from the initial viewport. Also, I wanna see if he will make another pr to bring it back lmao.

**Review by r2dev2 — CHANGES_REQUESTED** [review]
Other than feedback, looks good!

Just mentioning this here cuz I just thought of it: a lot of the images are jpg, it would be ideal to use webp as it has much better compression than png and is lossless. I think a lot of network bandwith can be saved, but this seems like a future issue.

**Comment by AVDestroyer** [comment]
murj

**Review by r2dev2 — APPROVED** [review]
